internal/rpcproxy/rpcproxy.go

```diff
@@ -735,6 +740,20 @@ func (p *clientProxy) handleAdminFacade(ctx context.Context, msg *message) (clie
 
 		return controllerLoginMessageFnc(user)
 	case "Login":
+		var request params.LoginRequest
+		err := json.Unmarshal(msg.Params, &request)
+		if err != nil {
+			return errorFnc(err)
+		}
+		user, err := names.ParseUserTag(request.AuthTag)
+		if err != nil {
+			return errorFnc(fmt.Errorf("invalid user tag: %v", err))
+		}
+		if user.Name() == api.AnonymousUsername {
```

Comment:
and empty domain?

Reply:
Good catch, switched to using `user.Id()` which will contain the domain